src/test/java/de/caritas/cob/consultingtypeservice/api/controller/TopicControllerIT.java

```diff
@@ -80,6 +82,18 @@ void getTopicList_Should_ReturnTopicsList_When_UserIsAuthenticated()
         .andExpect(jsonPath("$[0].createDate").exists());
   }
 
+  @Test
+  void getAllActiveTopics_Should_returnActiveTopicsList() throws Exception {
```

Comment:
in the Controller-Level we do not have Tenant-Aware-Tests.
How can I introduce this? Because I think this is missing. I added some for the repository, and may also in the Service. But here I am not sure. I think the Tenant Think I still not understood ;)

Reply:
you need to create a separate Tenant-Aware class for Test. You can then @Autowire the bean in test and set with ReflectionTestUtils the multitenancy flag to true. Please remember to clear the flag after the tests.